File: src/main/java/com/optica/domain/City.java
package com.optica.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class City {

	@Id
	@GeneratedValue
	private Long id;
	
	private String cityName;
	
	private Boolean mainCity;
	
	private Long latitude;
	
	private Long length;
	
	
	

}

File: src/main/kotlin/com/optica/domain/City.kt
package com.optica.domain

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
class City {

    @Id
    @GeneratedValue
    private val id: Long? = null

    private val cityName: String? = null

    private val mainCity: Boolean? = null

    private val latitude: Long? = null

    private val length: Long? = null


}
